src/Components/Header/Header.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import './Header.css';
+
+export default function Header() {
+  return (
+    <header>
+      <NavLink className={(isActive) => `${isActive ? 'active' : ''}`} to="/rgb/255/0/0">
```

Comment:
this solution for the class name is for when you want to add a class to non-active links, so in this case it's superfluous. I believe NavLinks automatically have an 'active' class you can access